store/sqlstore/session_store.go

```diff
@@ -135,6 +135,45 @@ func (me SqlSessionStore) GetSessionsWithActiveDeviceIds(userId string) ([]*mode
 	return sessions, nil
 }
 
+func (me SqlSessionStore) GetSessionsExpired(thresholdMillis int64, mobileOnly bool, unnotifiedOnly bool) ([]*model.Session, *model.AppError) {
+	query :=
```

Comment:
Please move it to use the squirrel query builder given that we're moving everything to use it :)

Reply:
Ugh. Ok, I'll read up on how to use this furry rodent.

Reply:
Done